### ISSUE TITLE
Let descriptions method operate on entire namespaces

### ISSUE THREAD

**Issue by elevans** [scijava/scijava#136]
**Context**: Jython script

Suppose I want to run a gaussian blur on an `net.imglib2.img.Img` but I don't know the op signature `filter.gauss`. Let's say that I know that the gaussian blur op is in the `filter` namespace but I'm unsure if its `gauss` or `gaussian`. In this scenario it would be great to have a mechanism to return all ops in a given namespace.

Current behavior:

```python
# get op environment
from org.scijava.ops.api import OpEnvironment
ops = OpEnvironment.getEnvironment()

print(ops.descriptions("filter"))
```
Output:
```python
[]
```

Proposed behavior:
```python
# get op environment
from org.scijava.ops.api import OpEnvironment
ops = OpEnvironment.getEnvironment()

print(ops.descriptions("filter"))
```
Output:
```python
[filter.addNoise(), filter.bilateral(), filter.convolve(), filter.gauss(), filter.hessian(), filter.ifft(), filter.max(), ...]
```

With this behavior I can easily find that the op signature I want in the `filter` namespace is `filter.gauss()`. I don't think we need to return all the possible input parameters for the ops as that will create an unruly list. Instead I think just returning the signature is fine. The user can get the description of the specific op for parameter information if they desire.

**Comment by ctrueden** [comment]
For more advanced cases like substring searching, you could do `ops.descriptions().stream().filter(...)`. I.e., let's not make the descriptions function too clever.

**Comment by ctrueden** [comment]
Another possibility would be to support the wildcard symbol `*` to match as a glob.

**Comment by gselzer** [comment]
Here's my opinions:
* `ops.descriptions()` returning all Ops would be too much to read. What if we just return the namespace?
```
ops.help()
>> namespaces: ["create", "filter", ...]
>> names: [
```
Implementation-wise, I think that the following is the way to go:
```java
Set<String> set = new HashSet<>();
for (String s : opNames) {
  String prefix = s.substring(s.indexOf('.')); // if no dot then the whole thing
  set.add(prefix)
}
```
* `ops.descriptions(String name)` will return:
  * First, each name contained within the namespace, like @elevans suggested in the initial comment.
  * Below that, all Ops named `name`. 
```
ops.help("create")

>> namespaces: ['create.img', 'create.type', ... ]
>> create( input: RAI<.....)
```

We probably also want to return a single string, not a `List<String>`.

**Comment by gselzer** [comment]
Closed by scijava/incubator#95
